### ISSUE TITLE
New Update QUESTIONS :)

### ISSUE THREAD

**Issue by C-odes** [swiety85/angular2gridster#323]
Hey! @swiety85  Thanks for the update! Fixed the scroll issue. But I have questions.

I wrote npm install angular2gridster  and then changed app-root{}  to demo-root{}  in the css.

However I notice you changed more files. Such as
projects/angular2gridster/src/lib/gridster.component.ts
projects/angular2gridster/src/lib/utils/utils.ts

I wanted to make sure the changes are there in my project so I went into  node_modules file. 
Found angular2gridster and searched for these files.

I only found the declare files of the following:
projects/angular2gridster/src/lib/utils/utils.ts
projects/angular2gridster/src/lib/gridster.component.ts 

only the declare files.
Does that sound correct or is something wrong? 

I just want to make sure I have the update fully and learn how these things work :)

Best Regards,
C-odes

**Comment by swiety85** [comment]
Hi, I'm not sure if I understand you right?
You should not have any `projects` directory there. There should be no `.ts` file.
I suggest you remove `node_modules` and call `npm i` again.

**Comment by C-odes** [comment]
Hey.
node_modules is outside my projects. I just wanted to double check npm install angular2gridster added everything (but I dont think I can see the ts files anyways so my bad.) 

only thing I can see is .d.ts files.  IS this wrong? :

![image](https://user-images.githubusercontent.com/42934761/51825803-dbae5000-22e5-11e9-8894-69ca63597deb.png)

d.ts files look like this: 

![image](https://user-images.githubusercontent.com/42934761/51825829-e9fc6c00-22e5-11e9-95c9-a657aa045e66.png)

Im starting to be pretty sure I just didnt understand how this works properly 😄 

**Another question. Is it now possible to have a max length for the components to be dragged? So one is not able to drag it too far down?**

Thanks again for the quick fix. This came at the nick of time!

**Comment by swiety85** [comment]
No, for now, there is no way to limit the gridster with a specific height, but there is already an issue for that: #287
